### ISSUE TITLE
New creatures as upgrades to standard creatures

### ISSUE THREAD

**Issue by Warzyw647** [vcmi-mods/kurek-creatures#5]
Consider making your creatures upgrades of in-town creatures. For example Cyclops King could have 2 alternative upgrades (upgradable in Hill Forts): Bloodeyed Cyclopes and Cyclops Warriors.
That way they will be able to appear more often in games.
This would also let them appear on my Arena once I add a Hill Fort to it sometime soon in vcmi 1.6.

**Comment by Kurekw** [comment]
No, sorry, I don't want to debalance the game too much. These creatures are unique, they also are meant to be equal to the Cyclops Kings, not  stronger upgrade.
